cases/services/news_enricher.py

```
@@ -0,0 +1,1519 @@
+    text = " ".join(parser.text_parts)
+    text = re.sub(r"\s+", " ", text).strip()
+    return text
+
+
+def _extract_images_from_html(html: str, base_url: str = "") -> list[dict]:
+    """Extract content-relevant image URLs from HTML, filtering out non-editorial images."""
+    parser = _ImageExtractor(base_url=base_url)
+    try:
+        parser.feed(html)
+    except Exception:
+        pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)